compute/k8s-services/vars.tf

```diff
@@ -52,6 +52,12 @@ variable "eks_cluster_name" {
   type = string
 }
 
+variable "eks_is_sandbox" {
```

Comment:
This new variable is not needed, because we can assert the same from an existing data source:

data.terraform_remote_state.cluster.outputs.eks_is_sandbox